tests/test_libzim.py

```diff
@@ -184,3 +184,11 @@ def test_double_close(tmpdir):
     creator.close()
     with pytest.raises(RuntimeError):
         creator.close()
+
+
+def test_default_creator_params(tmpdir):
+    """ ensure we can init a Creator without specifying all params """
+    creator = Creator(str(tmpdir / "test.zim"))
+    assert True  # we could init the Creator without specifying other params
```

Comment:
I think in is unnecessary. If you can't init the Creator, it will fail before.

Reply:
Yes, I just though it looked unfinished without it :) docstring makes it clear anyway.